src/apis/axios/index.ts

```diff
@@ -0,0 +1,19 @@
+import axios, { AxiosRequestConfig } from 'axios';
+
+const DEFAULT_TIMEOUT = 30000;
```

Comment:
요런식으로 표현해주시는 게 좋습니다~
```suggestion
const DEFAULT_TIMEOUT = 30_000;
```

Reply:
그런데 30초는 너무 길지 않을까요?

Reply:
아직 지정된 값이 아닌 임시 값입니다. 추후 수정하겠습니다.